packages/orbit-components/src/SmartPassIllustrations/illustrations/getHeight.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import theme from "../../defaultTheme";
+import { SIZE_OPTIONS } from "../../primitives/IllustrationPrimitive/consts";
+import type { Sizes } from "..";
+
+const getHeightToken = (size: Sizes) => {
+  const tokens = {
+    [SIZE_OPTIONS.EXTRASMALL]: theme.orbit.heightIllustrationSmall,
+    [SIZE_OPTIONS.SMALL]: "120px",
+    [SIZE_OPTIONS.MEDIUM]: theme.orbit.heightIllustrationMedium,
+    [SIZE_OPTIONS.LARGE]: "280px",
+    [SIZE_OPTIONS.DISPLAY]: "460px",
+  };
+  return tokens[size];
+};
+
+export default getHeightToken;
```

Comment:
It would be better to declare `tokens` as a constant in `packages/orbit-components/src/SmartPassIllustrations/illustrations/IllustrationWrapper.js`, then get the height simply with `tokens[size]`, I don't see a reason to have a `getHeightToken` function, and to split this logic in a separate file (the fewer files, the better).